[Release notes] Cloud changelogs catch up (#960)
seqeralabs/docs: changelog/seqera-cloud/v25.3.0_cycle33.md

date: 2025-11-25
tags: [seqera cloud]
---

## General updates and improvements

- **Studios**
  - Updated the **Add Studio** page to a single-step form.
- AWS Batch compute environments now use AmazonLinux2023-based AMIs by default, offering improved security (IMDSv2), better performance with gp3 EBS volumes, and a more recent kernel. Existing compute environments continue to function with AmazonLinux2 without changes, though we recommend migrating to newer compute environments before AmazonLinux2 reaches end-of-life in June 2026.

## Bug fixes

- Fixed an issue where compute environment variables could be created without specifying a target environment (head or compute), which would result in variables not being applied during workflow execution. The system now validates that at least one target environment is specified, preventing invalid configurations and providing clear error messages.
- Fixed an issue where long helper text in Nextflow pipeline schema forms would extend beyond component boundaries, causing display issues. Helper text now wraps properly within form controls, improving readability for parameter descriptions with technical specifications or long value lists.

> - Fixed a security vulnerability where GitHub tokens and other sensitive credentials were being exposed in user API responses through an undocumented options field. This field has been removed from all user information endpoints to prevent credential leakage.

Reviewer: ```suggestion
- Fixed a security vulnerability exposing GitHub tokens and other sensitive credentials in user API responses. The undocumented `options` field that could be exploited has been removed from all user information endpoints.
```

Were full GH tokens and credentials being exposed? According to our docs elsewhere, everything is encrypted at rest, so we should be very sure this is the case before announcing so, as this would be a serious security vulnerability if entirely true. cc @seqeralabs/security @tcrespog
